pkg/apis/networking/v1alpha1/ingress_validation.go

```diff
@@ -137,6 +138,11 @@ func (b IngressBackend) Validate(ctx context.Context) *apis.FieldError {
 	var all *apis.FieldError
 	if b.ServiceNamespace == "" {
 		all = all.Also(apis.ErrMissingField("serviceNamespace"))
+	} else if b.ServiceNamespace != apis.ParentMeta(ctx).Namespace {
+		all = all.Also(&apis.FieldError{
+			Message: "Ingress namespace is different with service",
```

Comment:
```suggestion
			Message: "service namespace must match ingress namespace",
```

(lowercase seems to be usual for these messages)

Reply:
Done